src/core/wagmi/createWagmiClient.ts

```diff
@@ -21,47 +22,55 @@ const noopStorage = {
   removeItem: () => null,
 };
 
+const getOriginalRpcEndpoint = (chain: Chain) => {
+  switch (chain.id) {
+    case ChainId.hardhat:
+      return { http: chain.rpcUrls.default.http[0] };
+    case ChainId.mainnet:
+      return { http: process.env.ETH_MAINNET_RPC as string };
```

Comment:
can't you type it as string in `.env.d.ts`? not sure how it works

Reply:
Good call!